src/Microsoft.Fx.Portability/CompressedHttpClient.cs

```diff
@@ -14,22 +14,38 @@
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Security.Authentication;
 using System.Threading.Tasks;
 
 namespace Microsoft.Fx.Portability
 {
     internal class CompressedHttpClient : HttpClient
     {
+        internal const SslProtocols SupportedSSLProtocols = SslProtocols.Tls12;
+
+#if FEATURE_SERVICE_POINT_MANAGER
+        internal const SecurityProtocolType SupportedSecurityProtocols = SecurityProtocolType.Tls12;
+#endif
+
         /// <param name="productName">Product name that will be displayed in the User Agent string of requests</param>
         /// <param name="productVersion">Product version that will be displayed in the User Agent string of requests</param>
         public CompressedHttpClient(ProductInformation info)
-            : this(info, new HttpClientHandler { AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate })
+            : this(info, new HttpClientHandler {
```

Comment:
Do we need this overload? We may be able to remove it to minimize duplication. We're already injecting a handler in the ApiPortService.

Reply:
Both overloads are used, currently, depending on whether `ApiPortService` is constructed with an `IProxyProvider` or not. The one without the proxy provider is only used by tests and SeachFxApi, so I'm sure the code could be changed to only need one overload but that might belong in another PR.

Reply:
Ok. We can remove not a blocker